neuralprophet/forecaster.py

```diff
@@ -811,6 +813,18 @@ def predict(self, df, decompose=True, raw=False):
             forecast = pd.concat((forecast, fcst), ignore_index=True)
         df = df_utils.return_df_in_original_format(forecast, received_ID_col, received_single_time_series)
         self.predict_steps = self.n_forecasts
+        # Conformal prediction interval with q
+        if self.q_hats:
+            if self.conformal_method == "naive":
+                df["yhat1 - qhat1"] = df["yhat1"] - self.q_hats[0]
```

Comment:
I think it would be more understandable to the general user, if we keep uncertainty naming consistent and also just show one uncertainty value in the df. 
As we currently use `f"yhat1 {quantile_lo}%"` and  `f"yhat1 {quantile_hi}%"` I suggest we stick with those.

Reply:
CP method is naive, so it does not take into account quantile low and quantile high. That is for CQR. Naive is only for `yhat` +- q, so saying `f"yhat1 {quantile_lo}%"` in incorrect here.